ada_2_fhir/mp/9.0.7/ant-build.log

```diff
@@ -1,68 +1,8 @@
 Buildfile: C:\Users\144189-ADM\Documents\Git\HL7-mappings\ada_2_fhir\mp\9.0.7\_ant-buildfiles\ant-publish\build-ada2fhir-mp-907.xml
```

Comment:
Maybe not that important (and probably unrelated to the specific change made in this PR), but why is a log file part of the PR?

Reply:
You could argue that it shouldn't be indeed. Maybe add it to .gitignore.

On the other hand, this repository is kind of a mess, so sometimes it actually helps me to compare log files to identify unwanted changes.